### PR TITLE
Add zsh-emojis to plugin list

### PR DESCRIPTION
**Pull request by MichaelAquilina**
I wrote a very simple zsh plugin that populates numerous ascii art emojis as variables in your environment. 

I initially used to store these myself in my `.zshrc` file, but some friends expressed interest in having something like this available to them. So I decided to make it into a simple plugin ᕙ(⇀‸↼‶)ᕗ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,6 +320,7 @@ These frameworks make customizing your zsh setup easier.
 * [zsh-directory-history](https://github.com/tymm/zsh-directory-history) - A per directory history for zsh.
 * [zsh-dwim](https://github.com/oknowton/zsh-dwim) - zsh-dwim attempts to predict what you will want to do next. It provides a key binding (control-u) that will replace the current (or previous) command line with the command you will want to run next.
 * [zsh-editing-workbench](https://github.com/psprint/zsh-editing-workbench) - Adds sane, complex command line editing (e.g. incremental history _word_ completion).
+* [zsh-emojis](httsp://github.com/MichaelAquilina/zsh-emojis.git) - Adds numerous ascii art emojis to your environment in convenient variables.
```
Comment (unixorn) [inline README.md]: change `httsp` to `https` and then I can merge.

Comment (MichaelAquilina) [inline README.md]: doh, thanks!

**Comment by MichaelAquilina** [comment]
@unixorn should be good to go now

**Comment by unixorn** [comment]
Thanks for the contribution!

**Pull request merged**